lms_homework/No28/classes.py

```diff
@@ -0,0 +1,218 @@
+class Node:
+    def __init__(self, data, next=None):
+        self.data = data
+        self.next = next
+
+
+class MyUnorderedList:
+
+    def __init__(self, first_element=None):
+        self.pointer = Node(first_element)  # Creat first Node then push in Queue
+
+        if first_element is None:
+            self.lenght = 0
+        else:
+            self.lenght = 1
+
+    def __str__(self):
```

Comment:
Think how you can simplify this method. It can be implemented in 5-6 lines of code actually.